goat.sh

```diff
@@ -37,6 +38,28 @@ get_entry() {
 
 }
 
+#Adds relative mapping to a file
+add_relative() {
```

Comment:
Either this function should replace the existing relative shortcut if needed or goat should try every relative shortcuts with the supplied name before giving up.